src/bindings/python/src/pyopenvino/core/tensor.cpp

```diff
@@ -133,6 +133,14 @@ void regclass_Tensor(py::module m) {
             py::arg("type"),
             py::arg("shape"));
 
+    cls.def(py::init<ov::Output<ov::Node>>(), py::arg("node_port"));
+
+    cls.def(py::init([](ov::Output<ov::Node>& port, py::array& array) {
+                return ov::Tensor(port, const_cast<void*>(array.data(0)));
+            }),
+            py::arg("node_port"),
```

Comment:
```suggestion
    cls.def(py::init<ov::Output<ov::Node>>(), py::arg("port"));

    cls.def(py::init([](ov::Output<ov::Node>& port, py::array& array) {
                return ov::Tensor(port, const_cast<void*>(array.data(0)));
            }),
            py::arg("port"),
```
Now I feel like keeping port would be OK... let's align it.

Reply:
updated